### DISCUSSION TITLE
Welcome to dracut Discussions!

### DISCUSSION THREAD

**Discussion by haraldh** [dracutdevs/dracut#994] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by johannbg** [comment]
Hi

I'm Jóhann B. Guðmundsson my main area of focus in the project is doing some gardening work, hopefully clean up the CI, better integrate the initramfs infrastructure with the system and service management framework ( systemd ) some documentation and whatever else that might fall my way and I encourage everyone to find his or hers area of interest in the Dracut project and start contributing.
